sass/_navigation.scss

```diff
@@ -5,223 +5,226 @@
 5.1 Links
 --------------------------------------------------------------*/
 a {
-    border-bottom: 1px solid transparent;
-	color: $link-color;
-	text-decoration: none;
-	transition: all .2s ease-in-out;
-
-	b &,
-    strong & {
-        border-width: 2px;
+  border-bottom: 1px solid transparent;
+  color: $link-color;
+  text-decoration: none;
+  transition: 
+      border .2s ease-in-out,
+      color .2s ease-in-out,
+      opacity .2s ease-in-out;
+
+  b &,
+  strong & {
+    border-width: 2px;
+  }
+
+  &:visited {
+    color: $link-color;
+  }
+
+  &:hover,
+  &:focus,
+  &:active {
+    color: $link-color-active;
+  }
+
+  &:hover {
+    color: $link-color-active;
+    text-decoration: none;
+
+    .entry-summary &,
+    .entry-content &,
+    .edit-link &,
+    .entry-meta & {
+      border-color: $link-border-color;
     }
+  }
 
-	&:visited {
-        color: $link-color;
+  &.author {
+    .name {
+      border-bottom: 1px solid transparent;
+      transition: border .2s ease-in-out;
     }
 
-	&:hover,
-    &:focus,
-    &:active {
-        color: $link-color-active;
-    }
-
-	&:hover {
-        color: $link-color-active;
-		text-decoration: none;
+    &:hover {
+      border-color: transparent;
 
-		.entry-summary &,
-        .entry-content &,
-        .edit-link &,
-        .entry-meta & {
-            border-color: $link-border-color;
-        }
+      .name {
+        border-color: $link-border-color;
+      }
     }
-
-	&.author {
-		.name {
-			border-bottom: 1px solid transparent;
-			transition: border .2s ease-in-out;
-		}
-
-		&:hover {
-			border-color: transparent;
-
-			.name {
-				border-color: $link-border-color;
-			}
-		}
-	}
+  }
 }
 
 /*--------------------------------------------------------------
 5.2 Menus
 --------------------------------------------------------------*/
 .main-navigation {
-	clear: both;
-	display: block;
-	float: left;
-	width: 100%;
-
-	ul {
-        list-style: none;
-        margin: 0;
-        padding-left: 0;
-
-        &.hide {
-            display: none;
-        }
-    }
-
-    a {
-        border-bottom: 1px solid #ddd;
-        display: block;
-        padding: 10px 0;
-        text-decoration: none;
+  clear: both;
+  display: block;
+  float: left;
+  width: 100%;
+
+  ul {
+    list-style: none;
+    margin: 0;
+    padding-left: 0;
+
+    &.hide {
+      display: none;
     }
+  }
+
+  a {
+    border-bottom: 1px solid #ddd;
```

Comment:
Color literals like `#ddd` should only be used in variable declarations; they should be referred to via variable everywhere else.